### PR TITLE
Add feature that enables adding an empty line at the bottom

### PR DESCRIPTION
**Pull request by AndrewKeepCoding**

### CODE REVIEW

File: source/INI/PeanutButter.INI/INIFile.cs [hunk 1]
```diff
@@ -1058,6 +1066,7 @@ PersistStrategies persistStrategy
             var addSeparator = SectionSeparator is not null;
             var commentDelimiter = lineParser.CommentDelimiter;
             var sep = CommentIfNecessary(SectionSeparator, commentDelimiter);
+            var addEmptyLineAtTheBottom = AddEmptyLineAtTheBottom;
```
Comment (fluffynuts) [inline source/INI/PeanutButter.INI/INIFile.cs:1069]: no need to var off - just use the property inline at the end - I don't think INIFile is rated for thread safety anyway 😄

File: source/INI/PeanutButter.INI/INIFile.cs [hunk 2]
```diff
@@ -28,6 +28,11 @@ public interface IINIFile
         /// </summary>
         string SectionSeparator { get; set; }
 
+        /// <summary>
+        /// Toggle whether an empty line is added at the bottom when persisted
+        /// </summary>
+        public bool AddEmptyLineAtTheBottom { get; set; }
```
Comment (fluffynuts) [inline source/INI/PeanutButter.INI/INIFile.cs:34]: I think I'd prefer the name `AppendTrailingEmptyLine`. Your thoughts?

**Review by fluffynuts — CHANGES_REQUESTED** [review]
Two minor suggestions, but otherwise, looks fine

File: source/INI/PeanutButter.INI/INIFile.cs (hunk 1 above)
Comment (AndrewKeepCoding) [reply source/INI/PeanutButter.INI/INIFile.cs]: Yeah, I agree. I was just trying to fit in the style.😁

File: source/INI/PeanutButter.INI/INIFile.cs (hunk 2 above)
Comment (AndrewKeepCoding) [reply source/INI/PeanutButter.INI/INIFile.cs]: English is not my native language, so let's go with your suggestion.👍

**Review by fluffynuts — COMMENTED** [review]
LGTM

**Review by fluffynuts — APPROVED** [review]

**Comment by AndrewKeepCoding** [comment]
Thanks!

**Pull request merged**